rspec-mode.el

```diff
@@ -146,6 +146,11 @@ Not used when running specs using Zeus or Spring."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-zeus-sock-file (concat (rspec-project-root) ".zeus.sock")
```

Comment:
it seemed when I tested it that this was evaluated when I entered rspec-mode, and at that time spec-project-root was valid.  But you're right, just using the environment variable is better, I'll change the method to get the path from $ZEUSSOCK first, then check for .zeus.sock in the project root.  This was basically just a "this worked for me and might be useful" first attempt at the PR.  I'll try and update it sometime today.

I didn't do that initially because I didn't think the variable was set where I was launching my editor on the vagrant vm, but I just hadn't reloaded my shell after successfully provisioning.